vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -94,77 +104,76 @@ You can use the default roles by clicking on `Client Roles` and choosing `entand
 
-* **Create & Assign Projects**
-1. From the landing page, click on any customer to see the associated project list.
-2. To create a new project, click `Add a Project`
-    * Enter the `Project` information
-    * Provide the Organization ID from Jira. Each project must have a unique Organization ID.
-    * Click `Save`
+* **Create and Assign Projects**
+1. From the CP landing page, click on any customer to see the associated project list.
+2. To create a new project: 
+   * Click `Add a Project`.
+   * Enter the `Project` information.
+   * Provide the Organization ID from Jira. Each project must have a unique Organization ID.
+   * Click `Save`.
 3. To assign `Projects`:
-	* Use the drop-down menu under `Action` on the right → `Manage Users`
-    * Select the user for the `Project`  → `Submit` 
+	 * From the `Action` drop-down menu →  `Manage Users`
+    * Select the user for the `Project` 
+    * Click `Submit` 
     
 * **Manage Partners and Subscriptions** \
-Use the drop-down menu under `Action` to manage Partners, and request and manage subscriptions. 
+Use the `Action` drop-down menu to manage Partners, and request and manage subscriptions. 
 
 ## Using the Customer Portal
 
-Customer Portal users need access provided by your organization administrator. You can request and track subscriptions as well as create and track service tickets.
+To access the Customer Portal, your organization administrator needs to provide  you with the proper user credentials. You can then create and track service tickets as well as request subscriptions.
 
-Once you login to the Customer Portal, you will see a list of customers. Click on any customer to see their details and track their projects.
+Once you login to the Customer Portal, you will see a list of customers. Click on a customer to view details and track their projects.
 
-1. To create a `Service Ticket` for a project, use the `Action` pull-down menu  → `Open Ticket`
+1. To create a `Service Ticket` for a project: \
+from the `Action` pull-down menu → `Open Ticket`
 
 ![Customer Portal Open Ticket](./images/cp-open-ticket.png)
 
-2. To request a Subscription or track Tickets, use the `Action` pull-down menu and select the corresponding option.
+2. To request a Subscription or track Tickets: \
+use the `Action` pull-down menu and select the corresponding option.
 
 ## Resources
 * Go to [Jira Service Management](https://www.atlassian.com/software/jira/service-management) for more information.
  
 
-* Source Code\
-The source for the Entando Customer Portal can be found with our other open source examples and tutorials on GitHub at: [https://github.com/entando-samples/customer-portal/](https://github.com/entando-samples/customer-portal/)
+* The **source code** for the Entando Customer Portal can be found with our other open source examples and tutorials on GitHub at: [https://github.com/entando-samples/customer-portal/](https://github.com/entando-samples/customer-portal/)
```

Comment:
instead of bolding "source code" would have that be the name of the link:

"The [source code](https://github.com/entando-samples/customer-portal/) for the Entando Customer Portal can be found with our other open source examples and tutorials on GitHub."